src/js/components/TaskDetail.js

```diff
@@ -226,22 +289,28 @@ class TaskSidePanelContents extends SidePanelContents {
     }
 
     return (
-      <div className="container-fluid container-pod container-pod-short flush-top">
+      <div className="container container-fluid flush">
+        <div className="media-object-spacing-wrapper container-pod container-pod-super-short flush-top">
+          <div className="media-object">
+            {this.getResources(task)}
+          </div>
+        </div>
         <DescriptionList
-          className="container container-fluid container-pod container-pod-short flush-bottom"
+          className="container container-fluid flush container-pod container-pod-super-short flush-top"
           hash={headerValueMapping}
           headline="Configuration" />
         <DescriptionList
-          className="container container-fluid container-pod container-pod-short flush-bottom"
+          className="container container-fluid flush container-pod container-pod-super-short flush-top"
           hash={labelMapping}
           headline="Labels" />
+        {this.getMarathonTaskDetailsDescriptionList()}
```

Comment:
Still I think the `task` should be passed consistently to keep the code consistent 

`getMarathonTaskDetailsDescriptionList(marathonTask)`
...
`getResources(mesosTask)`